zzzz_modular_occulus/code/game/objects/items/weapons/traitor_weapons.dm

```diff
@@ -0,0 +1,231 @@
+		return
+
+	unwield(U)
+	U.swap_hand() // For whatever reason the grab will not properly work if we don't have the free hand active.
+	var/obj/item/weapon/grab/G = M.grabbedby(U, 1)
+	U.swap_hand()
+	if(G && istype(G))
+		if(improvised) // Improvised garrotes start you off with a passive grab, but keep you stunned like an agressive grab.
+			M.Stun(1)
+		else
+			G.state = GRAB_NECK
+			G.hud.icon_state = "kill"
+			G.hud.name = "kill"
+			M.silent = max(M.silent, 10)
+
+	garrote_time = world.time + 10
+	START_PROCESSING(SSobj, src)
+	strangling = M
+	update_icon()
+
+	playsound(src.loc, 'sound/weapons/cablecuff.ogg', 15, 1, -1)
+
+	M.visible_message("<span class='danger'>[U] comes from behind and begins garroting [M] with the [src]!</span>", \
+				  "<span class='userdanger'>[U] begins garroting you with the [src]![improvised ? "" : " You are unable to speak!"]</span>", \
+				  "You hear struggling and wire strain against flesh!")
+
+	return
+
+/obj/item/twohanded/garrote/Process()
+	if(!strangling)
+		// Our mark got gibbed or similar
+		update_icon()
+		STOP_PROCESSING(SSobj, src)
+		return
+
+
+	if(!istype(loc, /mob/living/carbon/human))
+		strangling = null
+		update_icon()
+		STOP_PROCESSING(SSobj, src)
+		return
+
+	var/mob/living/carbon/human/user = loc
+	var/obj/item/weapon/grab/G
+
+	if(src == user.r_hand && istype(user.l_hand, /obj/item/weapon/grab))
+		G = user.l_hand
+
+	else if(src == user.l_hand && istype(user.r_hand, /obj/item/weapon/grab))
+		G = user.r_hand
+
+	else
+		user.visible_message("<span class='warning'>[user] loses their grip on [strangling]'s neck.</span>", \
+				 "<span class='warning'>You lose your grip on [strangling]'s neck.</span>")
+
+		strangling = null
+		update_icon()
+		STOP_PROCESSING(SSobj, src)
+
+		return
+
+	if(!G.affecting)
+		user.visible_message("<span class='warning'>[user] loses their grip on [strangling]'s neck.</span>", \
+				"<span class='warning'>You lose your grip on [strangling]'s neck.</span>")
+
+		strangling = null
+		update_icon()
+		STOP_PROCESSING(SSobj, src)
+
+		return
+
+	if(G.state < GRAB_NECK) // Only possible with improvised garrotes, essentially this will stun people as if they were aggressively grabbed. Allows for resisting out if you're quick, but not running away.
+		strangling.Stun(3)
+
+	if(improvised)
+		strangling.stuttering = max(strangling.stuttering, 3)
+		strangling.apply_damage(2, OXY, "head")
+		return
+
+
+	strangling.silent = max(strangling.silent, 10) // Non-improvised effects
+	strangling.apply_damage(4, OXY, "head")
+
+/obj/item/weapon/pen/edagger
+	icon = 'zzzz_modular_occulus/icons/obj/weapons.dmi'
+	origin_tech = "combat=3;syndicate=1"
+	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut") //these wont show up if the pen is off
+	var/on = 0
+	var/brightness_on = 2
+	light_color = "#B40000"
+
+/obj/item/weapon/pen/edagger/attack_self(mob/living/user)
+	if(on)
+		on = 0
+		force = initial(force)
+		sharp = 0
```

Comment:
`FALSE`

Reply:
Not resolved.

Reply:
AAA?! I swear I did. 😿

Reply:
No, I definitely did!

Reply:
![image](https://user-images.githubusercontent.com/12434413/155258897-a4485dd8-84c5-4615-8660-6eece24e0a63.png)

Reply:
`sharp = FALSE`

Reply:
Fixed.